File: Python Exams/Basic Exam Collection/Basics Online Exam - 20 and 21 April 2019/06. Easter Competition/06. Easter Competition.py
number_players = int(input())
max_points = 0
winner = ''

while number_players:
    number_players -= 1
    current_points = 0
    chef_name = input()
    while True:
        try:
            current_points += int(input())
        except:
            break

    print(f'{chef_name} has {current_points} points.')

    if max_points < current_points:
        max_points = current_points
        winner = f'{chef_name} won competition with {max_points} points!'
        print(f'{chef_name} is the new number 1!')


print(winner)



'''
2
Chef Angelov
9
9
9
Stop
Chef Rowe
10
10
10
10
Stop
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 20 and 21 April 2019/06. Easter Competition/06. Easter Competition.js
function easterCompetition(arg) {

    let numberPlayers = arg.shift()
    let maxPoints = 0
    let winner = ''

    while (numberPlayers) {
        numberPlayers -= 1
        let currentPoints = 0
        let chefName = arg.shift()
        while (true) {
            let input = +arg.shift()

            if (isNaN(input)) {
                console.log(`${chefName} has ${currentPoints} points.`)
                break
            } else {
                currentPoints += +input
            }
        }

        if (maxPoints < currentPoints) {
            maxPoints = currentPoints
            winner = `${chefName} won competition with ${maxPoints} points!`
            console.log(`${chefName} is the new number 1!`)
        }
    }

    console.log(winner)
}

easterCompetition([2, 'Chef Angelov', 9, 9, 9, 'Stop', 'Chef Rowe', 10, 10, 10, 10, 'Stop'])